Funcky.DiscriminatedUnion.Test/SourceGeneratorTest.cs

```diff
@@ -51,12 +61,11 @@ private static CSharpCompilation CreateCompilation(params SyntaxTree[] syntaxTre
         => CSharpCompilation.Create(
             nameof(SourceGeneratorTest),
             syntaxTrees,
-            new[]
-            {
+            [
                 MetadataReference.CreateFromFile(Assembly.Load("System.Runtime").Location),
                 MetadataReference.CreateFromFile(typeof(Attribute).Assembly.Location),
                 MetadataReference.CreateFromFile(typeof(Console).Assembly.Location),
-                MetadataReference.CreateFromFile(typeof(System.Text.Json.JsonSerializer).Assembly.Location),
-            },
+                MetadataReference.CreateFromFile(typeof(System.Text.Json.JsonSerializer).Assembly.Location)
```

Comment:
no trailing comma anymore?

Reply:
good catch, resharper must've removed it when converting to collection literals